msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,621 @@
+---
+title: Add an Outlook Add-in to a Teams app
+description: Learn how to add an Outlook Add-in to a Teams app. This article will walk you through the steps to add an Outlook Add-in to a Teams app, test it, then deploy it to Azure.
+ms.date: 02/15/2024
+ms.author: mosdevdocs
+author: rickki
+ms.topic: tutorial
+ms.localizationpriority: medium
+ms.subservice: m365apps
+---
+
+# Add an Outlook Add-in to a Teams app
+
+## Introduction
+
+[Outlook Add-ins](/office/dev/add-ins/outlook/outlook-add-ins-overview) are web apps that extend the functionality of Outlook. With an Outlook Add-in, you can:
+
+- Read and write email messages and engage in meeting invitations, responses, cancellations, and appointments.
+- Read properties of the user's mailbox.
+- Respond to events automatically, such as sending an email.
+- Integrate with external services including CRM and project management tools.
+- Add custom ribbon buttons or menu items to perform specific tasks.
+
+For more info, [Outlook Add-ins Overview](/office/dev/add-ins/outlook/outlook-add-ins-overview) provides information about what you can do with Outlook Add-ins, how they are structured, how they differ from older ways of extending Outlook, what platforms the add-in can run on (Windows, Mac, mobile, and the web), and how to get started creating one.
+
+Combining an Outlook Add-in with a Teams app enables scenarios that can't be satisfied with either of them separately. For example, consider an app that enables a salesperson to insert a customized discount in an email to a customer and records the offer in a backend database. The sales manager can also view data about all the discounts that have been offered in a Teams tab. For more information, see [Discount Offers sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-add-in-combined/nodejs).
+
```

Comment:
Integrating an Outlook Add-in with a Teams app provides unique functionalities that can't be fulfilled individually. Consider a scenario where a salesperson can insert a customized discount in an email to a customer and record the offer in a backend database, all within Teams. The sales manager can then view data about all the discounts that have been offered in a Teams tab. For more information, see [Discount Offers sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-add-in-combined/nodejs).

[Outlook Add-ins Overview](/office/dev/add-ins/outlook/outlook-add-ins-overview) provides insights on the capabilities of Outlook Add-ins, their architecture, improvements over previous extension methods, the platforms they support (Windows, Mac, mobile, and web), and steps to begin developing your own.

Reply:
> Consider a scenario where a salesperson can insert a customized discount in an email to a customer and record the offer in a backend database, all within Teams. The sales manager can then view data about all the discounts that have been offered in a Teams tab. For more information, see [Discount Offers sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-add-in-combined/nodejs).
> 
> [Outlook Add-ins Overview](/office/dev/add-ins/outlook/outlook-add-ins-overview) provides insights on the capabilities of Outlook Add-ins, their architecture, improvements over previous extension methods, the platforms they support (Windows, Mac, mobile, and web), and steps to begin developing your own.

These changes are not accurate. This would NOT be "all within Teams".

Reply:
Can we consider moving Line 26 to Line 24 and bring Line 24 below?

I have also rephrased without much change in meaning:

Integrating an Outlook Add-in with a Teams application creates unique scenarios that neither of them could achieve independently. Consider a scenario where a salesperson can insert a customized discount in an email to a customer and record the offer in a backend database. The sales manager can also view data about all the discounts that have been offered in a Teams tab. For more information, see [Discount Offers sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-add-in-combined/nodejs).

[Outlook Add-ins Overview](https://github.com/office/dev/add-ins/outlook/outlook-add-ins-overview) provides insights on the capabilities of Outlook Add-ins, how they are structured, how they are improved over previous extension methods, what platforms the add-in can run on (Windows, Mac, mobile, and the web), and how to get started creating one.

Reply:
OK. I've made this change, with slight variations in wording.